frontend/src/components/Home/ItemCard.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import { Typography, Grid, Button, CardHeader, CardMedia } from '@mui/material';
```

Comment:
```suggestion
import { Typography, Grid, Button, CardHeader } from '@mui/material';
```
nit: unused import

Reply:
ItemCard should've been removed from the last commit. Done now.